pending/change-session-id.pod

```diff
@@ -0,0 +1,107 @@
+=head1 change_session_id
+
+It is considered good security practice to change session ID on any change
+of privilege level, for example on login.  It is also a requirement of a
+number of established security standards so is something you should be
+doing in your web applications.
+
+=head2 Session destruction
+
+Destroying a session in L<Dancer2> has always been possible using
+L<Dancer2::Core::App/destroy_session> thus:
+
+    app->destroy_session;
+
+Session destruction by its very nature causes the session ID to change since
+the server instructs the client to forget the session ID, send it a new
+session ID and also destroys any session data held on the server.  This is
+considered a "good thing" when a user logs out of an application but it is
+not very useful when a user logs in and you want to keep existing session
+data such as cart products.
+
+=head2 change_session_id
+
+L<Dancer2::Core::App/change_session_id> to the rescue!
+
+If you are using a recent version of L<Dancer2::Plugin::Auth::Extensible>
+to handle authentication and authorisation then you already have
+C<change_session_id> called on successful login. If you are using some
+other solution then you want to do something like this:
+
+    post '/login' => sub {
+        my $username = body_parameters->get("username");
+        my $password = body_parameters->get("password");
+
+        if ( my_authentication_check( $username, $password ) ) {
+            # login successful
+            app->change_session_id;
+        }
+    };
+
+You might also want to consider calling C<change_session_id> after successful
+password change.
+
+=head2 The gory details
+
+If you are planning to write a new session engine you should really read
+this part...
```

Comment:
I'd rather have something like "This part is particularly relevant to implementors of session engines".